pages/data.py

```diff
@@ -101,7 +113,30 @@ def render_content(tab, store_uuids, store_trips, store_demographics, store_traj
 
     return populate_datatable(df)
 
+# handle subtabs for demographic table when there are multiple surveys
+@callback(
+    Output('subtabs-demographics-content', 'children'),
+    Input('subtabs-demographics', 'value'),
+    Input('store-demographics', 'data'),
+)
+
+def update_sub_tab(tab, store_demographics):
+    data = store_demographics["data"]
+    if tab in data:
+        data = data[tab]
+        
+        if data:
+            columns = list(data[0].keys())
+            has_perm = perm_utils.has_permission('data_demographics')
```

Comment:
@achasmita as we discussed, I don't see how setting this here affects anything in the main tab. you should remove this line. you might need to move the `has_perm` check in `render_context` before the `if` to compensate.